### PR TITLE
Modifying sendTo_Ether contract + step by step guide for learners to follow

### LINKED ISSUE #53: Modifying sendTo_Ether contract
### Description of  the change

Modifying and bringing new changes to https://github.com/Vikash-8090-Yadav/Solidity-Pathshala/tree/main/Level1/sendTo_Ether

This currently has these problems ->
1. No step by step guide for new users to use the smart contract.
2. Balance returns in default Wei which is 12 digit and quite confusion to new users.
3. Does not cover the end cases i.e. what if the contract has 0 balnce or the contract has more balance than to be transferred or both have equal balance.

### Domain 

- [x] Readme
- [x] Solidity
- [] Python
- [] JavaScript
- [] Golang
- [] Other , feel free to mention below

#### Are you contributing under any open-source program ?
SSOC22
<hr/>

### PR DESCRIPTION
**Pull request by legendarykamal**
## Description
Modifying sendTo_Ether contract + step by step guide for learners to follow in the README.md file added.

Fixes #53 
## Type of change

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this :

- [] Add a new project

- [] Modified or enhanced the existing projects

- [] Improve the readme of any project

## Domain of Contribution

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [x] Project using only Solidity

- [] Project using only Javascript

- [] Project using only Python

- [x] Project Test on the Remix IDE

- [] Mention below if project uses other language and ide .

## Checklist ✓

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [x] l have performed a self-review of my own code or work.

- [x] I have commented on my code, particularly in hard-to-understand areas.

- [] My changes generate no new warnings

- [] I'm HSOC'22 contributor

- [x] I'm SSOC'22 contributor

### CODE REVIEW

File: Level1/sendTo_Ether/Readme.md [hunk 1]
```diff
@@ -36,17 +36,30 @@ And here you can see the all the transactions and deployed contracts-
...
+![Screenshot from 2022-08-24 13-43-42](https://user-images.githubusercontent.com/95926324/186368582-d4912d60-6796-4c6f-bb79-5cbb0feee437.png)
+
 and you can replace this by the one declared as friend address in the contract just to verify :)
 
 4> Now click on -> 
+
+![Screenshot from 2022-08-24 13-45-05](https://user-images.githubusercontent.com/95926324/186368642-86059fd2-5a95-45ce-bcce-97be43c9d18a.png)
+
 This shall transfer 5 ethers out of 10 from your contract to your choosen account and you shall have something like this in the transaction status.
+![Screenshot from 2022-08-24 13-47-39](https://user-images.githubusercontent.com/95926324/186368727-f47e40f5-c1fa-4801-981c-357dbdb974ea.png)
```
Comment (Vikash-8090-Yadav) [inline Level1/sendTo_Ether/Readme.md:63]: explain the use  of  each function in the readme

Comment (legendarykamal) [reply Level1/sendTo_Ether/Readme.md:72]: @Vikash-8090-Yadav done :+1

**Comment by legendarykamal** [comment]
// @Vikash-8090-Yadav

**Comment by Vikash-8090-Yadav** [comment]
Great @legendarykamal

**Pull request merged**